src/python/ddapp/ik.py

```diff
@@ -369,3 +369,61 @@ def tick(self):
 
         if self.handleAsyncTasks() > 0:
             return
+            
+    def runMultiRRT(self, grHand, qStart, xGoal, objectGrasped):
```

Comment:
runMultiRRT() is currently the entry point for this new capability. It has two steps:
* planner.findFinalPose() - find a collision free end pose which can be used instead of runIK
* planner.findCollisionFreeTraj(xGoalFull) - which is the main rrt planner

As you can see it now mirrors runIkTraj(). On another PR I will merge it into runIkTraj().